features-json/push-api.json

```diff
@@ -227,7 +227,7 @@
       "35":"n"
     },
     "and_chr":{
-      "49":"a #1"
+      "50":"y"
```

Comment:
I updated the commit so that it only has one version. The CI check is still failing.